src/collections/_documentation/platforms/javascript/react.md

```diff
@@ -146,3 +146,135 @@ class App extends React.Component {
 
 export default App;
 ```
+
+## Redux
+
+{% version_added 5.20.0 %}
+
+Redux support is included in the `@sentry/react` package since version `5.20.0`. To use redux you would use `Sentry.createReduxEnhancer` at the same place that you initialize your Redux store.
```

Comment:
```suggestion
Redux support is included in the `@sentry/react` package since version `5.20.0`. To apply Redux, use `Sentry.createReduxEnhancer` at the same place that you initialize your Redux store.
```